Codeinwp/themeisle-content-forms
# Content Forms Improvements

**What's new:**

- Better organize the code and files
- Fixed the issues with the required fields ( https://github.com/Codeinwp/themeisle-companion/issues/338 )
- Fixed the issue with the Beaver Modules registering multiple times ( https://github.com/Codeinwp/themeisle-companion/issues/384 )
- Added the possibility to change the success or error messages for contact and newsletter widgets ( https://github.com/Codeinwp/themeisle-content-forms/issues/35 )
- Added the possibility to register with the newsletter widget not only the email and name but also the other fields that MailChimp or SendinBlue allows
- Added the option to hide all form labels
- Added the styling controls for beaver widgets ( almost the same as Elementor forms have )

**How to install it:**

1. Get the development branch of Orbit Fox: https://github.com/Codeinwp/themeisle-companion/tree/development
2. Run composer install in a terminal
3. Replace the themeisle-content-forms folder in vendor/codeinwp/ with this one ( the dev branch of this repo )

**Testing suggestions:**

1. Check that all style options are working ( I've tested them but maybe I missed something )
2. Check the mobile part. ( I haven't had time to test it but I will too )
